r1cs-std/src/fields/fp12.rs

```diff
@@ -834,6 +834,17 @@ where
     P: Fp12Parameters,
     <P::Fp6Params as Fp6Parameters>::Fp2Params: Fp2Parameters<Fp = ConstraintF>,
 {
+    #[inline]
+    fn alloc_constant<T, CS: ConstraintSystem<ConstraintF>>(
+        mut cs: CS,
+        t: T,
+    ) -> Result<Self, SynthesisError>
+    where
+        T: Borrow<Fp12<P>>,
+    {
+        Self::zero(cs.ns(|| "zero"))?.add_constant(cs.ns(|| "add constant"), t.borrow())
```

Comment:
Is there a reason not to just allocate each coordinate individually, besides this just being shorter?

Reply:
Just wanted to benefit from the `add_constant` convenience, it should be equivalent. Wdyt?

Reply:
Sure, sounds good to me =)